admin/views/identity-provider/creation.ejs

```diff
@@ -95,6 +95,28 @@
               </div>
             </div>
 
+            <%
+              let isEntraIDChecked = false
+              if (locals.messages.values && messages.values[0].isEntraID !== undefined) {
+                isEntraIDChecked = messages.values[0].isEntraID
+              } else {
+                isEntraIDChecked = false
+              }
+            %>
+            <div class="mb-3 row">
+              <label class="col-2 col-form-label">Identité portée par le ID token :</label>
+              <div class="mb-3 col-4 d-flex align-items-end">
+                <div class="form-check form-check-inline">
+                    <input type="radio" class="form-check-input is-valid" id="entra" name="isEntraID" value="true" required <%= isEntraIDChecked ? 'checked' : '' %>>
+                    <label class="form-check-label" for="tokenCentric">oui</label>
```

Comment:
nitpick: this and `userInfoCentric` could be updated to use more consistent naming.

The same remark applies to the `update.ejs` template.